File: Android/java/doraemonkit-no-op/src/main/java/com/didichuxing/doraemonkit/kit/network/okhttp/interceptor/LargePictureInterceptor.java
package com.didichuxing.doraemonkit.kit.network.okhttp.interceptor;



import java.io.IOException;

import okhttp3.Interceptor;
import okhttp3.Request;
import okhttp3.Response;

/**
 * 大图拦截器
 */
public class LargePictureInterceptor implements Interceptor {

    @Override
    public Response intercept(Chain chain) throws IOException {
        Request request = chain.request();
        return chain.proceed(request);
    }

}

File: Android/kotlin/doraemonkit-no-op/src/main/java/com/didichuxing/doraemonkit/kit/network/okhttp/interceptor/LargePictureInterceptor.kt
package com.didichuxing.doraemonkit.kit.network.okhttp.interceptor

import okhttp3.Interceptor
import okhttp3.Response
import java.io.IOException

/**
 * 大图拦截器
 */
class LargePictureInterceptor : Interceptor {
    @Throws(IOException::class)
    override fun intercept(chain: Interceptor.Chain): Response {
        val request = chain.request()
        return chain.proceed(request)
    }
}